modules/backend/widgets/Lists.php

```diff
@@ -926,7 +926,7 @@ protected function makeListColumn($name, $config)
             $config['searchable'] = false;
         }
 
-        $columnType = $config['type'] ?? null;
+        $columnType = $config['type'] ?? '';
```

Comment:
Should this default to text instead?

Reply:
the issue is i'm going around the entirety of winter and there are a lot of things that are giving the warnings with no context  as to what is "using" the internal functions, just that the functions are being used and null is being passed to them.

Reply:
So is this PR still a work in progress then?

Reply:
yes, there's still a lot of areas that I have to go through, I also have to do a pr for winter storm as well.

Reply:
are you not getting these warnings on php 8.1?

Reply:
I'm not running any projects on 8.1 at the moment, wanted to focus more on the Laravel 9 specific issues rather than have to deal with both at the same time.